lib/elixir_newbie/blog/blog.ex

```diff
@@ -0,0 +1,12 @@
+defmodule ElixirNewbie.Blog do
+  alias ElixirNewbie.Blog.Post
+
+  use NimblePublisher,
+    build: Post,
+    from: Application.app_dir(:elixir_newbie, "priv/posts/**/*.md"),
+    as: :posts,
+    highlighters: [:makeup_elixir, :makeup_erlang]
```

Comment:
We can remove the :makeup_erlang highlighter here since we removed it as a dependency.